src/attributes.js

```diff
@@ -70,6 +70,22 @@ const applyProp = function(el, name, value) {
 
 
 /**
+ * Applies a value to a style declaration. Supports CSS custom properties by
+ * setting properties containing a dash using CSSStyleDeclaration.setProperty.
+ * @param {CSSStyleDeclaration} style
+ * @param {!string} prop
+ * @param {*} value
+ */ 
+const setStyleValue = function(style, prop, value) {
+  if (prop.indexOf('-') >= 0) {
+    style.setProperty(prop, /** @type {string} */(value));
```

Comment:
I wanted to do that initially, but setProperty does not work with the JavaScript convention (camelCase). The question is should we still support the camelCase version. We could test to see if there are any capital letters and then use the property, would have to test it to see if it the performance is just as good.